platforms/minestom/build.gradle.kts

```diff
@@ -0,0 +1,31 @@
+plugins {
+    application
+}
+
+val javaMainClass = "com.dfsek.terra.minestom.TerraMinestomExample"
+
+dependencies {
+    shadedApi(project(":common:implementation:base"))
+    shadedApi("com.github.ben-manes.caffeine", "caffeine", Versions.Libraries.caffeine)
+    shadedImplementation("com.google.guava", "guava", Versions.Libraries.Internal.guava)
+
+    implementation("net.minestom", "minestom-snapshots", Versions.Minestom.minestom)
+    implementation("org.slf4j", "slf4j-simple", Versions.Libraries.slf4j)
+}
+
+tasks.withType<Jar> {
+    entryCompression = ZipEntryCompression.STORED
+    manifest {
+        attributes(
+            "Main-Class" to javaMainClass,
+        )
+    }
+}
+
+application {
+    mainClass.set(javaMainClass)
+}
+
+tasks.getByName("run").setProperty("workingDir", file("./run"))
+
+addonDir(project.file("./run/terra/addons"), tasks.named("run").get())
```

Comment:
afaik somewhere you need to depend on the `installAddons` task.

see: https://github.com/PolyhedralDev/Terra/blob/0a952cff4c3eea8bc0a49205643e0c225d12304c/platforms/fabric/build.gradle.kts#L47-L55

Reply:
Should be fixed by the latest commit